docusaurus/docs/shared/extension-points.md

```diff
@@ -0,0 +1,4 @@
+| Type          | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| ------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Link**      | Links have a `path` and an `onClick()` property. <br /><br /> **When to use?** <br /> Use links if you would like to give plugins a way to define custom user actions for a part of your UI. These actions can either just be cross-links to the plugin, or using `onClick()` methods they can implement a more interactive on-page experience with a modal. <br /><br /> **API reference** <br /> - [`addLink()`](../../reference/ui-extensions.md#addlink) - registering a link from a plugin <br /> - [`usePluginLinks()`](../../reference/ui-extensions.md#usepluginlinks) - fetching links registered for an extension point |
```

Comment:
All of the links in this shared file are broken. I believe they can be fixed by removing the ".md" between the file ID and the anchor.

Reply:
fixed